share/picongpu/examples/LaserWakefield/lib/python/picongpu/ui_params.py

```diff
@@ -0,0 +1,20 @@
+from eupraxia_picongpu.parameters import *
```

Comment:
REQUIRED:
Please add a license header to this file.

Reply:
we should add a list of self-contained classes here, so the GUI repo will depend on PIConGPU but not vice versa.

Reply:
location: `lib/python/picongpu/input/`

Reply:
from which directory on would the include then be needed?
Like this
```python
from picongpu.input.parameters import *
```
or some directory above/below?